polkadot/runtime/westend/src/lib.rs

```diff
@@ -598,12 +650,12 @@ impl pallet_bags_list::Config<VoterBagsListInstance> for Runtime {
 
 pallet_staking_reward_curve::build! {
 	const REWARD_CURVE: PiecewiseLinear<'static> = curve!(
-		min_inflation: 0_025_000,
-		max_inflation: 0_100_000,
-		ideal_stake: 0_500_000,
-		falloff: 0_050_000,
-		max_piece_count: 40,
-		test_precision: 0_005_000,
+		min_inflation: 0_025_000, // dynamic_params::staking_reward_curve::MinInflation,
+		max_inflation: 0_100_000, // dynamic_params::staking_reward_curve::MaxInflation::get(),
+		ideal_stake: 0_500_000, // dynamic_params::staking_reward_curve::IdealStake::get(),
+		falloff: 0_050_000, // dynamic_params::staking_reward_curve::Falloff::get(),
+		max_piece_count: 40, // dynamic_params::staking_reward_curve::MaxPieceCount::get(),
+		test_precision: 0_005_000, // dynamic_params::staking_reward_curve::TestPrecision::get(),
```

Comment:
@kianenigma to clarify, is this what you meant by making the inflation formula parameterizable? Or does the ticket actually require the implementation of `pallet_inflation`?

Reply:
This is on-point! 

What I would request further is for you to make the `type EraPayout` in westend use exactly what is proposed here: 

https://github.com/polkadot-fellows/runtimes/pull/364/files#diff-efa4caeb17487ecb13d8f5eb7863c3241d84afa2e73fbf25909a2ca89df0f362L667

You can start by moving the `fn relay_era_payout` to `runtime-common`, mark the old one as deprecated, and use the new one in westend and any other runtime here.

Reply:
Got it, I've marked the old function as deprecated for now, and added some unit test for the new one